### PR TITLE
Emphasize the need for a URL

### PR DESCRIPTION
**Pull request by Skorfulose**
## Proposed change

Emphasize the need for a URL (as opposed to IP/hostname only) in this integration's docs.
Without the "http://" specifier you'll receive an error on config check or Home Assistant restart.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/fronius.markdown [hunk 1]
```diff
@@ -20,14 +20,14 @@ To enable this sensor, add the following lines to your `configuration.yaml` file
 ```yaml
 sensor:
   - platform: fronius
-    resource: FRONIUS_URL
+    resource: http://FRONIUS_URL
```
Comment (frenck) [inline source/_integrations/fronius.markdown]: Please revert this, as this is a placeholder per our standards.

```suggestion
    resource: FRONIUS_URL
```

File: source/_integrations/fronius.markdown [hunk 2]
```diff
@@ -20,14 +20,14 @@ To enable this sensor, add the following lines to your `configuration.yaml` file
...
     monitored_conditions:
     - sensor_type: inverter
 ```
 
 {% configuration %}
 resource:
-  description: "The IP address of the Fronius device"
+  description: "The URL of the Fronius device (e.g. \"http://192.0.2.0\" or \"http://fronius.local\")"
```
Comment (frenck) [inline source/_integrations/fronius.markdown]:
```suggestion
  description: "The URL of the Fronius device (e.g., `http://192.0.2.0` or `http://fronius.local`)"
```

File: source/_integrations/fronius.markdown [hunk 3]
```diff
@@ -96,7 +96,7 @@ a list of sensors that are to be integrated can be given like below.
 ```yaml
 sensor:
   - platform: fronius
-    resource: FRONIUS_IP_ADDRESS
+    resource: http://192.0.2.0
```
Comment (frenck) [inline source/_integrations/fronius.markdown]:
```suggestion
    resource: FRONIUS_URL
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @Skorfulose!

I've reviewed it and left some suggestions, could you please take a look? Thanks!

**Review by Skorfulose — COMMENTED** [review]
Suggestions accepted 👍

**Review by frenck — APPROVED** [review]
Thanks, @Skorfulose 👍

**Pull request merged**